### ISSUE TITLE
VS Code in browser with GitHub makes 1000ths of copies (vscode.dev).

### ISSUE THREAD

**Issue by arthur-adriansens** [microsoft/vscode#212049]
Does this issue occur when all extensions are disabled?: **Yes**

- VS Code Version: https://vscode.dev/, version 1.89, April 2024
- OS Version: Google ChromeOS, Version 122.0.6261.118 (Official Build) (64-bit). I'm using my school's Chromebook and working with the files from Google Drive and GitHub.

Steps to Reproduce:

1. Open your Chromebook, go to vscode.dev and open it as an app (the bug also occurs if I just use vscode.dev and don't use the app).

2. Open a GitHub repository and make a new file. Make some edits to the file, or any file really, and with every autosave, your Google Drives main folder will be full of thousands of useless copies of the files. Google Drive makes it really difficult to remove so many files. I couldn't use my chromebook for at least 15 minutes while deleting only a small portions of the files:

![Screenshot 2024-05-05 15 15 07](https://github.com/microsoft/vscode/assets/79015591/742c6201-37f0-44ed-90e0-d887566c3552)

**Comment by arthur-adriansens** [comment]
Hi everyone! Did someone find the issue? I still experience the same thing...

**Comment by rzhao271** [comment]
Hi Arthur, I assume the issue comes from vscode.dev's autosave combined with using Google Drive to store files. In step 2, when you say GitHub repository, are you creating a git repository on Google Drive and then opening that repository as a folder?

**Comment by arthur-adriansens** [comment]
> Hi Arthur, I assume the issue comes from vscode.dev's autosave combined with using Google Drive to store files. In step 2, when you say GitHub repository, are you creating a git repository on Google Drive and then opening that repository as a folder?

Yes, I assume that causes the issue too. In step 2, I just open an **excisting** remote GitHub repository, without cloning.

**Comment by arthur-adriansens** [comment]
Hi! The problem stil exists. My Chromebook might be the problem, because everything on it is stored on Google Drive.
